paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/activity/LaunchPaymentSheetCompleteActivity.kt

```diff
@@ -1,41 +1,212 @@
 package com.stripe.android.paymentsheet.example.activity
 
 import android.os.Bundle
-import androidx.core.view.isInvisible
+import android.widget.Toast
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.stripe.android.paymentsheet.PaymentSheet
-import com.stripe.android.paymentsheet.example.databinding.ActivityPaymentSheetCompleteBinding
+import com.stripe.android.paymentsheet.example.R
 
-internal class LaunchPaymentSheetCompleteActivity : BasePaymentSheetActivity() {
-    private val viewBinding by lazy {
-        ActivityPaymentSheetCompleteBinding.inflate(layoutInflater)
-    }
+val EMOJI_FONT_SIZE = 36.sp
```

Comment:
should we use companion object?

Reply:
There is not really a way with composables as they aren't in a class.

Reply:
Can they be a companion object of `LaunchPaymentSheetCompleteActivity`? I don't have a strong preference.

Reply:
I am going keep it here for now, many of these are things that should be a part of a theme, when we get that far.